backend/core/package.json

```diff
@@ -41,12 +41,12 @@
     "@anchan828/nest-sendgrid": "^0.3.25",
     "@google-cloud/translate": "^6.2.6",
     "@nestjs/cli": "^8.2.1",
-    "@nestjs/common": "^8.3.1",
+    "@nestjs/common": "9.3.7",
     "@nestjs/config": "^1.2.0",
-    "@nestjs/core": "^8.3.1",
+    "@nestjs/core": "9.3.7",
     "@nestjs/jwt": "^8.0.0",
-    "@nestjs/passport": "^8.2.1",
-    "@nestjs/platform-express": "^8.3.1",
+    "@nestjs/passport": "9.0.3",
+    "@nestjs/platform-express": "9.3.9",
     "@nestjs/schedule": "^2.1.0",
     "@nestjs/swagger": "5.2.0",
     "@nestjs/throttler": "^2.0.0",
```

Comment:
Github won't let me comment on the correct line, but shouldn't this PR be upgrading `typeorm` on line 85?